services/src/apidoc.rs

```diff
@@ -108,14 +113,22 @@ use utoipa::{Modify, OpenApi};
         handlers::upload::upload_handler
     ),
     components(
+        responses(
+            UnsupportedMediaTypeForJsonResponse,
+            PayloadTooLargeResponse,
+            IdResponse,
+            UnauthorizedAdminResponse,
+            UnauthorizedUserResponse,
+            BadRequestQueryResponse
+        ),
         schemas(
+            ErrorResponse,
```

Comment:
why is this unser schemas and not under response?

Reply:
see above